comm/src/main/scala/coop/rchain/comm/protocol.scala

```diff
@@ -5,6 +5,10 @@ import coop.rchain.comm.protocol.routing._
 import scala.util.{Failure, Success, Try}
 import scala.concurrent.duration.{Duration, MILLISECONDS}
 
+sealed trait ProtocolError extends CommError
```

Comment:
It is a good practice to define all elements of an ADT in one place, so we could use `sealed` modificator and easily observe all elements in one place.